archlinux/PKGBUILD-qubes-repo-4.0.conf

```diff
@@ -1,2 +1,3 @@
 [qubes-r4.0]
-Server = http://olivier.medoc.free.fr/archlinux/current
+#Replace the server below with your own
+Server = https://neowutran.ovh/qubes/vm-archlinux/pkgs/
```

Comment:
Do you plan to maintain this repository? If so, please state it explicitly and provide a public key and also update info in template docs. Otherwise, at the very least comment out this line.

Reply:
Removed the references to my server

About https://github.com/QubesOS/qubes-core-agent-linux/pull/222#pullrequestreview-359468967 pretty much the first big project I try to contribute to, idk what are the good way to do a multi-branch pull request. But for this "master" pull request, I don't intend to do something more complex than cherry picking the commits in this one